aws/rust-runtime/aws-types/src/sdk_config.rs

```diff
@@ -313,23 +316,61 @@ impl Builder {
         self
     }
 
+    /// Set the credentials cache for the builder
+    ///
+    /// # Examples
+    /// ```rust
+    /// use aws_credential_types::cache::CredentialsCache;
+    /// use aws_types::SdkConfig;
+    /// let config = SdkConfig::builder()
+    ///     .credentials_cache(CredentialsCache::lazy())
+    ///     .build();
+    /// ```
+    pub fn credentials_cache(mut self, cache: CredentialsCache) -> Self {
+        self.set_credentials_cache(Some(cache));
+        self
+    }
+
+    /// Set the credentials cache for the builder
```

Comment:
We should consider linking here to types/docs that describe in greater depth how configuring credential caching works.

Reply:
Addressed in 1368f00.